### PR TITLE
create Hosted Widget customization examples

### PR DESCRIPTION
**Pull request by spencerlepine**
*Issue #, if available:*

*Description of changes:*
Creating `hostedWidgetCustomization` examples, showing customer how to personalize the widget on their website. 

This is paired with Hosted Widget icon improvement feature: https://docs.aws.amazon.com/connect/latest/adminguide/customize-widget-launch.html

## VIEW:
https://github.com/amazon-connect/amazon-connect-chat-ui-examples/tree/spenlep/hosted-widget-examples/hostedWidgetCustomization

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by xiajon-amzn — APPROVED** [review]

File: hostedWidgetCustomization/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      function () {
+        (w[x].ac = w[x].ac || []).push(arguments);
+      };
+  })(window, document, "amazon_connect", "<REPLACE_ME>");
+  amazon_connect("styles", {
+    openChat: { color: "#ffffff", backgroundColor: "#07b62a" },
+    closeChat: { color: "#ffffff", backgroundColor: "#07b62a" },
+  });
+  amazon_connect("snippetId", "<REPLACE_ME>");
+  amazon_connect("supportedMessagingContentTypes", [
```
Comment (doreechi) [inline hostedWidgetCustomization/README.md:28]: do we want to also add a note about rich messaging enabled vs disabled? Or just in general call out that other methods can be added here to pass display name, contact attributes, etc and add link?

File: hostedWidgetCustomization/loadAssetsOnButtonClick/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      skipIconButtonAndAutoLaunch: true,
+    });
+  });
+</script>
+```
+
+## Snippet Customizations
+
+```diff
+- <script type="text/javascript">
```
Comment (doreechi) [inline hostedWidgetCustomization/loadAssetsOnButtonClick/README.md:52]: can we update this diff to not include the script tags being removed? unless this is expected

File: hostedWidgetCustomization/README.md (hunk 1 above)
Comment (spencerlepine) [reply hostedWidgetCustomization/README.md:28]: Yes we can add follow up documentation for rich messaging (the Admin guide doesn't really explain that either) - however, customer cannot pass displayName/contact attributes in this snippet. 

If they build a custom widget, they can of course: https://github.com/amazon-connect/amazon-connect-chat-interface/blob/master/DOCUMENTATION.md#chat-interface-configuration

File: hostedWidgetCustomization/loadAssetsOnButtonClick/README.md (hunk 2 above)
Comment (spencerlepine) [reply hostedWidgetCustomization/loadAssetsOnButtonClick/README.md:52]: This is expected. Agree this isn't very clear

Customer needs to copy/paste **just** the code from this snippet, to use in their javascript code and they must remove the html tags

File: hostedWidgetCustomization/README.md (hunk 1 above)
Comment (doreechi) [reply hostedWidgetCustomization/README.md:28]: why can't they pass display name/contact attributes? is that specific to if they specify this auto launch behavior?

**Pull request merged**